mapbuilder/builder.py

```diff
@@ -38,10 +40,15 @@ def __init__(self, source_dir, target_dir, cache_dir, config):
                 self.data[data_source] = parser.parse()
             elif data_source_type == "raw":
                 logging.debug(f"Loading raw source {data_source}...")
-                with (source_dir / config["data"][data_source]["source"]).open(
-                    encoding="iso-8859-1",
-                ) as f:
-                    self.data[data_source] = f.read()
+                if os.path.isfile(source_dir / config["data"][data_source]["source"]):
+                    with (source_dir / config["data"][data_source]["source"]).open(
+                        encoding="iso-8859-1",
+                    ) as f:
+                        self.data[data_source] = f.read()
+                else:
+                    parser = RAWParser(source_dir / config["data"][data_source]["source"])
+                    self.data[data_source] = parser.parse()
+                    #logging.debug(self.data[data_source])
```

Comment:
Either remove debugging statements or leave them in if they provide any benefit.